config/routes.rb

```diff
@@ -114,9 +111,9 @@
     # settings
     get    '/settings' => 'users#edit', as: :edit_user
     patch  '/settings' => 'users/registrations#update'
-    get    '/settings/email' => 'users/registrations#edit_email', :as => :edit_user_email
-    get    '/settings/profile' => 'users/profiles#edit', :as => :edit_user_profile
-    get    '/settings/notifications' => "users/notifications#edit", :as => :settings_notifications
+    get    '/settings/email' => 'users/registrations#edit_email', as: :edit_user_email
+    get    '/settings/profile' => 'users/profiles#edit', as: :edit_user_profile
+    get    '/settings/notifications' => "users/notifications#edit", as: :settings_notifications
```

Comment:
Line is too long. [95/80]<br>Unnecessary spacing detected.<br>Put one space between the method name and the first argument.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.